requirements.txt

```diff
@@ -1,3 +1,8 @@
+certifi==2023.7.22
+charset-normalizer==3.2.0
+idna==3.4
+requests==2.31.0
+urllib3==2.0.5
```

Comment:
Looking at QueryAPI.py it seems that your only dependency here is `requests`

The others being sub-dependencies of `requests`

Please remove certifi, charset, idna, urllib3